### ISSUE TITLE
Unable to use keyboard navigation in Commit panel

### ISSUE THREAD

**Issue by alex-polic** [devlon-io/gitl#2]
When using commit panel, currently I am unable to:

1. Navigate the files with arrows
2. Click enter to open a file listed in changes
3. Pick files to commit via arrows and space (tabs work a bit, but clunky)
4. Navigate to Commit Message text area with tab quickly
